frontend/src/lib/components/CreateTable.spec.ts

```diff
@@ -0,0 +1,44 @@
+import {describe, it, expect, beforeEach, vi} from 'vitest';
```

Comment:
nitpick: sometimes you make spaces after `{` and sometimes not....

Reply:
true. In https://github.com/Zuretto/planning-poker/pull/7 it should be better.